bin/get_config_env.py

```diff
@@ -1,19 +1,31 @@
-import requests,json, os
+import json
+import os
+
+import requests
 from requests.auth import HTTPBasicAuth
 
-base_host = os.environ.get('DISCOVERY_HOST') or u'gateway.watsonplatform.net'
 
-request_url = "https://{0}/discovery-experimental/api/v1".format(base_host)
+VCAP_SERVICES = json.loads(os.environ.get('VCAP_SERVICES'))
+VCAP_DISCOVERY_CREDENTIALS = VCAP_SERVICES['discovery'][0]['credentials']
+VCAP_DISCOVERY_URL = VCAP_DISCOVERY_CREDENTIALS['url']
+VCAP_DISCOVERY_USERNAME = VCAP_DISCOVERY_CREDENTIALS['username']
+VCAP_DISCOVERY_PASSWORD = VCAP_DISCOVERY_CREDENTIALS['password']
+
+base_host = os.environ.get('DISCOVERY_HOST', VCAP_DISCOVERY_URL)
+
+request_url = "https://{0}/discovery/api/v1".format(base_host)
```

Comment:
the url has changed, why?

Reply:
i _assume_ you are correct. Discovery has been in production for many months now. I believe the original author used an Discovery product team example as his code base, so it's entirely possible he was using a URL he shouldn't have been.

Reply:
There is a v1-experimental client. It is old. Not sure what it gives you. Hopefully any old experimental features are in the regular client now. https://github.com/watson-developer-cloud/node-sdk/blob/master/discovery/v1-experimental.js#L42